clients/drcachesim/common/options.cpp

```diff
@@ -72,7 +72,8 @@ droption_t<std::string> op_ipc_name(
 droption_t<std::string> op_outdir(
```

Comment:
Given the rebranding of the framework as "drmemtrace" (because "drcachesim" is just one tool inside the framework), best to s/drcachesim/drmemtrace/ in the PR title and description.

Reply:
Except that these docs are specifically for drcachesim?

Reply:
No, these docs are not for the drcachesim simulator: they are for the drmemtrace framework's tracer and post-processor and front-end for running a variety of tools (just one of which is drcachesim).

Reply:
Interesting. The option is defined in clients/drcachesim/common/options.cpp. Okey doke.